docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -0,0 +1,698 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Qute Reference Guide
+
+include::./attributes.adoc[]
+
+:numbered:
+:sectnums:
+:sectnumlevels: 4
+:toc:
```

Comment:
I think we don't have that one in the other guides. The TOC is generated by the Jekyll plugin IIRC.

Reply:
Yep, I use this in the CDI ref guide to generate a numbered TOC. It seems to work fine.